SECURITY.md

```diff
@@ -0,0 +1,76 @@
+# OSM Security Process and Policy
+
+This document provides the details on the OSM security policy and details the processes
+surrounding security handling including a how to guide on reporting a security vulnerability
+for anything within the OSM organization.
+
+## Report A Vulnerability
+
+We’re extremely grateful for security researchers and users who report vulnerabilities
+to the OSM community. All reports are thoroughly investigated by a set of OSM maintainers.
+
+To make a report please email the private security list at cncf-openservicemesh-security@lists.cncf.io
+with the details.
+
+You may, but are not required to, encrypt your email to this list using the PGP keys
+of security team members, listed below.
+
+Name               | Key URL                                              | Fingerprint
+------------------ | ---------------------------------------------------- | -----------
+TBD                | TBD                                                  | TBD
+TBD                | TBD                                                  | TBD
+TBD                | TBD                                                  | TBD
+
+
+### When To Send A Report
+
+You think you have found a vulnerability in a OSM project or a dependency of a OSM project. This can be any of the repositories on the [OSM GitHub organization](https://github.com/openservicemesh).
+
+### When Not To Send A Report
+
+* If a vulnerability has been found in an application managed by OSM. Instead, contact the application maintainers
```

Comment:
I'm not sure "an application managed by OSM" is clear to me. Do we mean an application managed by an OSM control plane?

Reply:
```suggestion
* If a vulnerability has been found in a user application binary that does not belong to OSM, contact the application maintainers instead.
```